lib/haml_lint/tree/tag_node.rb

```diff
@@ -9,7 +9,7 @@ class TagNode < Node
     #
     # @return [Array<String>]
     def dynamic_attributes_sources
-      @value[:attributes_hashes]
+      @value.fetch(:attributes_hashes, [])
```

Comment:
@michaelherold true, seems like it was renamed (and it's data structure changed too?) https://github.com/haml/haml/commit/7c523f278a7c62eea249ea29c797bf29377e136e
Doing this fixed haml-lint for my codebase, but likely some checks are not working properly